package com.redhat.cleanbase.exception.func;

public interface FinallyFunc {
    void doAction() throws Exception;
}